src/renderer/components/outputs/StableDiffusionModelOutput.tsx

```diff
@@ -0,0 +1,76 @@
+import { literal } from '@chainner/navi';
+import { Center, Flex, Spacer, Text } from '@chakra-ui/react';
+import { memo, useEffect } from 'react';
+import { useContext, useContextSelector } from 'use-context-selector';
+import { struct } from '../../../common/types/util';
+import { isStartingNode } from '../../../common/util';
+import { BackendContext } from '../../contexts/BackendContext';
+import { GlobalContext, GlobalVolatileContext } from '../../contexts/GlobalNodeState';
+import { TypeTags } from '../TypeTag';
+import { OutputProps } from './props';
+
+interface StableDiffusionModelData {
+    arch: string;
+}
+
+export const StableDiffusionModelOutput = memo(
```

Comment:
This isn't anything you need to worry about, I just wanna say this would have been a whole lot easier if this was just a single TaggedOutput. I should probably open an issue for doing that

Reply:
I half-finished a type broadcast PR a few days ago. I should probably finish it. Then we wouldn't need those output kinds at all, since the python outputs could just declare a type broadcast, which would automatically be handled.

Reply:
@adodge Since #1681 is now merged, I believe all of these new frontend output components can be replaced by a tag type broadcast